### ISSUE TITLE
Stream output of `platform create`

### ISSUE THREAD

**Issue by ehmatthes** [ehmatthes/django-simple-deploy#125]
This is not as straightforward as some of the other streaming issues, #121 and #124. See comment below with code.

- [x] Stream output of `platform create` in `deploy_platformsh.py`.
- [x] Pass unit tests.
- [x] Pass integration tests.
- [x] Merge, release, test.

**Comment by ehmatthes** [comment]
Relevant code:

```
self.sd.write_output("  Running `platform create`...")
        self.sd.write_output("    (Please be patient, this can take a few minutes.")
        cmd = f'platform create --title { self.deployed_project_name } --org {self.org_name} --region {self.sd.region} --yes'

        try:
            # Include check=True in this call, so we can process more
            #   significant errors that can come out of this call.
            # For example, if user can't create a project the 
            #   returncode will be 6, not 1. That causes subprocess.run()
            #   to actually raise an exception, not just pass a CompletedProcess
            #   instance.
            output = self.sd.execute_subp_run(cmd, check=True)
        except subprocess.CalledProcessError as e:
            error_msg = plsh_msgs.unknown_create_error(e)
            raise CommandError(error_msg)

        self.sd.write_output(output)
```

**Comment by ehmatthes** [comment]
Working notes
---

- What exactly does `check=True` do?
    - `execute_subp_run()` is a wrapper in simple_deploy.py, around [subprocess.run()](https://docs.python.org/3/library/subprocess.html#subprocess.run). From docs:
```
If check is true, and the process exits with a non-zero exit code, a [CalledProcessError](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError) exception will be raised. Attributes of that exception hold the arguments, the exit code, and stdout and stderr if they were captured.
```

**Comment by ehmatthes** [comment]
Works from pypi, not currently
---

- [ ] Setting the git remote properly.
- [ ] checkout/ stash, then try again locally from original code.
- [ ] Works, after trying again without so much diagnostic code.

**Comment by ehmatthes** [comment]
Fixed by #186.
